Assignments/Sorting/InsertionSort.playground/Pages/Insertion Sort.xcplaygroundpage/Contents.swift

```diff
@@ -8,7 +8,22 @@ Note that there is already a `swap` function in the standard library. However, y
 */
 
 func insertionSort<T>(var array: [T], @noescape isOrderedBefore: (T, T) -> Bool) -> [T] {
-
+    guard array.count > 1 else {
```

Comment:
This is actually redundant as the for-loop will just never enter its body in the case of an empty or single item array.